smart_card_connector_app/src/chrome-api-provider.js

```diff
@@ -33,6 +33,9 @@ const PcscApi = GSC.PcscLiteClient.API;
 
 /** @type {!goog.log.Logger} */
 const logger = GSC.Logging.getScopedLogger('ConnectorApp.ChromeApiProvider');
+/** @type {number} Bitmask of the lower 16 bits. */
+const lowerBits = 0x0000FFFF;
```

Comment:
nit: Maybe some more informative name like `EVENT_FLAGS_MASK` or `EVENT_FLAGS_BITS`?

(And in caps, per style guide recommendation for "true constants".)